src/recorder.js

```diff
@@ -24,6 +24,7 @@ var Recorder = function( config ){
   this.config.encoderApplication = config.encoderApplication || 2049;
   this.config.encoderFrameSize = config.encoderFrameSize || 20;
   this.config.resampleQuality = config.resampleQuality || 3;
+  this.config.encoderComplexity = config.encoderComplexity || 9;
```

Comment:
I thought that you would prefer it this way cause its more consistent with the rest of the defaults. 

IMHO, it's better to have it explicitly because the opus docs describe it as on of the parameters to [tweak](https://wiki.xiph.org/OpusFAQ#What_is_the_complexity_of_Opus.3F) depending on your CPU. The values might be different for mobile / desktops, opus_fixed vs opus_float.

From the opus code, it looks like the default is 9, so that's the value I left it at https://github.com/xiph/opus/blob/master/src/opus_encoder.c#L224

BTW, I'll open a separate issue about performance in mobile, and whether there's a need for `_opus_encode_fixed` in addition to `_opus_encode_float`